components-starter/camel-aws2-ddb-starter/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1,21 @@
+## ---------------------------------------------------------------------------
```

Comment:
I do not think this file is needed in SB 2.x (camel 3.x)

Reply:
Re-tested again with this file deletion